components/Common/OramaSearchBar/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { create, load, search } from '@orama/orama';
+import { useTheme } from 'next-themes';
+import { useCallback, useMemo, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { OramaLogoDark, OramaLogoLight } from './OramaLogo';
+import styles from './index.module.scss';
```

Comment:
styles usually should be the last import 👀 

After all other imports (so before the `DocumentMapper` type

Reply:
I think I fixed it now. I let VS code fix it :)